backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -296,6 +296,35 @@ public Map<String, Object> getAttributes() {
         return attributes;
     }
 
+    /**
+     * Adds a new attribute to Backtrace Client. If the native integration is enabled, adds the attribute
+     * to the native report attributes if:
+     *  - the value exists (is not a null)
+     *  - is not an object (the attribute value is primitive type like String, or Int)
+     * @param key attribute name
+     * @param value attribute value.
+     */
+    public void addAttribute(String key, Object value) {
+        attributes.put(key, value);
```

Comment:
what if value will be null

Reply:
It's fine if the value is null. We want to store this information. We cannot pass it to the native